lib/AST/ASTImporter.cpp

```diff
@@ -6741,6 +6742,29 @@ Decl *ASTImporter::Import(Decl *FromD) {
   if (!FromD)
     return nullptr;
 
+  if (const auto *ND = dyn_cast<NamedDecl>(FromD))
+    if (IdentifierInfo *II = ND->getIdentifier()) {
+      if (II->isStr("__va_list_tag")) {
+        if (isa<TypedefDecl>(FromD) &&
+            ToContext.getTargetInfo().getBuiltinVaListKind() ==
+            TargetInfo::PowerABIBuiltinVaList) {
```

Comment:
There are many other kinds of VaLists see: https://clang.llvm.org/doxygen/classclang_1_1TargetInfo.html#a3502c200a07b649537ac95a3be7c8653

Does this work with all of them?